package.json

```diff
@@ -54,6 +54,7 @@
     "bunyan": "^1.0.0",
     "commander": "^2.13.0",
     "express": "^4.16.1",
+    "fs": "^0.0.1-security",
```

Comment:
Does this need to be in `dependencies`?  Looks like `claudia-api-builder`, which is used in the same file, is in `devDependencies`.